src/MSBuild/Resources/Strings.resx

```diff
@@ -787,6 +787,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_37_DocsLink" UESanitized="false" Visibility="Public">
+    <value>For more detailed information access the link:
```

Comment:
nit:
```suggestion
    <value>For more detailed information, see:
```

Reply:
```suggestion
    <value>For more detailed information, see https://aka.ms/msbuild/docs
```

? That shortlink points to `https://docs.microsoft.com/visualstudio/msbuild/msbuild` but can be easily moved if we need to later on.

Reply:
@rainersigwald Done!